integration-cli/docker_cli_diff_test.go

```diff
@@ -78,15 +75,13 @@ func (s *DockerSuite) TestDiffEnsureOnlyKmsgAndPtmx(c *check.C) {
 	}
 
 	for _, line := range strings.Split(out, "\n") {
-		if line != "" && !expected[line] {
-			c.Errorf("%q is shown in the diff but shouldn't", line)
-		}
+		c.Assert(line != "" && !expected[line], checker.Equals, false)
```

Comment:
I'm not very sure about this, what about when `line != ""` and ` expected[line]` happen?